docs/pages/workflow/customizing.mdx

```diff
@@ -5,21 +5,19 @@ description: Learn how to add custom native code to your Expo project.
+For most third-party native modules, thanks to [autolinking](/modules/autolinking/), all you need to do is install the package and create a new development build in order to start using the module's functionality. Some modules will also require you setup a [config-plugin](/config-plugins/introduction) to take care of other native customizations. For writing your own custom native code, there are different options depending on how you want to interact with native code in general.
 
-## Add custom native code with development builds
+## Create a local module with the Expo Modules API
 
-To use third-party libraries with custom native code and continue with the same developer experience of Expo Go, you can migrate to use [development builds](/develop/development-builds/introduction/).
+By default, Expo projects use [Continuous Native Generation (CNG)](/workflow/continuous-native-generation). This means that projects do not have **android** and **ios** folders containing the native code and configuration used to build the app. Instead, the native projects are generated just before they are built based on your [Expo Config](/workflow/configuration) native modules in **package.json**. This simplifies the management and upgrading of native code, but means you cannot edit the native code directly.
 
-Development builds are your personal version of Expo Go. They include the native runtime that powers your app. You control what is included in that native runtime by adding or removing libraries in the **package.json**. Development builds allow you to continue building your app in JavaScript while taking full advantage of the native libraries available for Expo and React Native projects.
+However, you can still add custom native code to a project using CNG by [creating a local Expo module](modules/get-started/#creating-the-local-expo-module). Local Expo Modules are just like [Expo Modules](/modules/overview/) created by package developers and those used within the Expo SDK itself, but they are not published to npm. Instead, they are created directly inside of your project. This allows you to write custom Swift and Kotlin code that you can interact with from your JavaScript coe, and is included in your development builds and production apps automatically when you build with [EAS Build](/build/introduction) or when you build locally with `npx expo run:[android|iOS]`.
 
-To learn how to start using custom native code in your app by switching from Go to development builds, see [create a development build](/develop/development-builds/create-a-build/).
+## Manage your own native projects
```

Comment:
```suggestion
## Manage custom native projects
```

I'd avoid using "your own" in section titles. We include section titles in search and "your" is not a phrase that is used by someone using search.